app/services/OverseasTransferChargesService.scala

```diff
@@ -39,7 +39,16 @@ class OverseasTransferChargesService @Inject()(pensionSessionService: PensionSes
       pensionIndex match {
         case Some(index) =>
           model.copy(transferPensionScheme = model.transferPensionScheme.updated(index, pensionScheme.copy(ukTransferCharge = Some(question))))
-        case None => model.copy(transferPensionScheme = model.transferPensionScheme ++ Seq(pensionScheme.copy(ukTransferCharge = Some(question))))
+        case None =>
+         if(model.transferPensionScheme.isEmpty) {
+          model.copy(transferPensionScheme = model.transferPensionScheme ++ Seq(pensionScheme.copy(ukTransferCharge = Some(question))))
```

Comment:
Also do you need to do `model.transferPensionScheme ++ Seq(pensionScheme.copy(ukTransferCharge = Some(question)))) ` becasue on this line `model.transferPensionScheme` is empty anyway

Reply:
Fixed.